src/State/ScreenProcessor.php

```diff
@@ -44,6 +54,43 @@ protected function fromInput(mixed $object, Operation $operation, array $uriVari
             $screen->setEnableColorSchemeChange($object->enableColorSchemeChange);
         }
 
+        if (isset($object->regionsAndPlaylists) && isset($screen)) {
```

Comment:
Would it be possible to have the same write and read API - `regionsAndPlaylists` is different from `regions`used in the read endpoint.